_data/master/navbars/networking.yml

```diff
@@ -4,10 +4,8 @@ order: 2
 toc:
 - title: Networking
   path: /networking/
-- title: Configuring IP-in-IP
```

Comment:
In addition to removing these docs from the sidebar, we should also remove the actual content of these docs from the git tree as well.

Reply:
Not sure what you mean. Can you tell me how to do this?

Reply:
You'll need to `git rm master/networking/ip-in-ip.md` and `git rm master/networking/vxlan.md` in order to remove the files from git - this PR as it is just removes them from the navigation bar, but the files are still "dangling" there.

Reply:
Ok. I'm removed them.